MIGRATION.md

```diff
@@ -88,6 +88,10 @@ Storybook now uses Babel 7. There's a couple of cases when it can break with you
     ```
   * If you're using Babel 6, make sure that you have direct dependencies on `babel-core@6` and `babel-loader@7`.
 
+### start-storybook opens browser automatically
+
+If you're using `start-storybook` on CI, you may need to opt out of this using the new `--ci` flag.
```

Comment:
Maybe check for `process.env.CI` too? It's set by most CI systems

Reply:
I actually think that `ci` is too limiting

Reply:
yep. it would be much better to have something like `BROWSER=none start-storybook`. This is the same, that is used by cra or docusaurus.